### ISSUE TITLE
learning_rate

### ISSUE THREAD

**Issue by GLaStu** [clxia12/RT-DETRv3#15]
When using the default configurations and parameters, the learning rate never updates. Could you tell me where to configure the learning rate update and how to do it?
